extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -759,13 +770,42 @@ export class ConsentUI {
         (this.restrictFullscreenOn_ &&
           this.document_.activeElement !== this.ui_)
       ) {
-        // TODO (@torch2424) Send response back if enter fullscreen was succesful
+        user().warn(TAG, FULLSCREEN_ERROR);
+        this.sendIframeMessage_(messageType.error, FULLSCREEN_ERROR);
         return;
       }
+      this.sendIframeMessage_(messageType.success, FULLSCREEN_SUCCESS);
 
       this.baseInstance_.mutateElement(() => {
         this.enterFullscreen_();
       });
     }
   }
+
+  /**
+   * Send messages to iframe (for error logging).
+   * Silently die if iframe does not have content
+   * window.
+   *
+   * Example message:
+   * {
+   *   type: 'type',
+   *   message: 'message',
+   * }
+   * @param {string} type
+   * @param {string} message
+   */
+  sendIframeMessage_(type, message) {
+    const iframeWindow = this.ui_.contentWindow;
+    if (iframeWindow) {
+      // No sensitive information sent, so safe to use '*'
+      iframeWindow./*OK*/ postMessage(
```

Comment:
Sorry I know we agree to send back the status and the error message, but I realize the format doesn't scale well.
Imagine we want to expand the feature to work with following request. 
```
{
  'type': 'consent-response'
  'action': 'accept'
  'info': '.............'
}
```

It would be difficult for CMPs to categorize response based on the message info. Instead I think the following items are also required. 
1. echo back the request type
2. echo back the request action
3. request status
4. errorMessage if there's any